databases/week1/hw.sql

```diff
@@ -0,0 +1,16 @@
+USE `hyf_class2023`;
+select count(*) from task;
+select count(*) from task WHERE due_date IS NULL; 
+select task.id as `TASK_ID`, task.title as `TITLE`, task.description  as `Description`,
+status.name as`STATUS_NAME` from task join status on status.id=task.status_id where status.name='Done';
+select task.id as `TASK_ID`, task.title as `TITLE`, task.description  as `Description`,
+status.name as`STATUS_NAME` from task join status on status.id=task.status_id where status.name!='Done';
+select * from task order by created ASC;
+SELECT  *   from task where created in (select MAX(created) as `Recently Created task`from task);
+# I am getting a row null value also not sure why please rectify
+
+select concat (task.title,'-',status.name) as `TEXT`from task join status on status.id=task.status_id;
```

Comment:
Would suggest as below instead, so you have a clear overview of the o/p

-- Get the title and status (as text) of all tasks
```
SELECT title,status.name as TEXT
FROM task
JOIN status ON task.status_id=status.id
ORDER BY status.name;
```

Reply:
Hi
I made those changes and pushed with new commit,please review,thank you.